### PR TITLE
Promote @boardfish and @Spone to maintainers

### PR DESCRIPTION
**Pull request by joelhawksley**
Welcome to the maintainers team, @spone and @boardfish ❤️

### CODE REVIEW

File: docs/CONTRIBUTING.md [hunk 1]
```diff
@@ -94,30 +94,20 @@ To run the Vale prose linter locally, `brew install vale` and `vale docs/`.
...
 
-ViewComponent is built by over a hundred members of the community. Project membership has three levels:
-
-### Triage
-
-ViewComponent triagers are able to manage issues and pull request by assigning owners and labels, closing issues and marking duplicates.
-
-After helping with the project by participating in issues, pull requests, and/or discussions, members of the community are welcome to request triage access by opening a pull request to update this list:
-
-The triagers team is @andrewmcodes, @bbugh, @boardfish, @cesariouy, @dark-panda, @dylnclrk, @g13ydson, @horacio, @jcoyne, @johannesengl, @kaspermeyer, @mellowfish, @metade, @nashby, @rainerborene, @rdavid1099, @spdawson, @yhirano55, and @vinistock.
```
Comment (boardfish) [inline docs/CONTRIBUTING.md:105]: Is the triage team being retired? I think if so, it'd still be good to encourage folks to get involved in issue triage even if there's no direct role attached to it. It was a good onramp for me to get involved in contributing to ViewComponent itself.

**Review by boardfish — APPROVED** [review]
Thanks so much! 🙏

File: docs/CONTRIBUTING.md (hunk 1 above)
Comment (joelhawksley) [reply docs/CONTRIBUTING.md:105]: That's a fair point. I removed it from the docs as none of the folks with `triage` have been active in that role. I'll re-add something 👍🏻

**Review by Spone — APPROVED** [review]
Thank you :)

**Pull request merged**